public/src/components/campaign/joined_campaigns.jsx

```diff
@@ -18,6 +18,14 @@ export default class JoinedCampaigns extends Component {
     const { first_name, last_name } = this.props.account_info;
     const page = 'Campaign';
     const is_admin = false;
+
+    if (
+      joined_campaigns.length !== 0 &&
+      Object.getOwnPropertyNames(current_campaign).length === 0
+    ) {
+      setCurrentCampaign(joined_campaigns[0]);
```

Comment:
I think this might need to go in componentDidMount

Reply:
Yeah I changed this to set the prop instead of mutating the store.